x/wasm/keeper/handler_plugin_encoders_test.go

```diff
@@ -515,3 +515,59 @@ func TestEncoding(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertWasmCoinToSdkCoin(t *testing.T) {
```

Comment:
This is a useful check to return errors early (at the cause), rather than some time later (worse if no one verifies again before storing it)